src/main/java/com/internet/shop/controllers/users/DeleteUsersController.java

```diff
@@ -0,0 +1,28 @@
+package com.internet.shop.controllers.users;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteUsersController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService =
```

Comment:
if you're making services final, make them final everywhere
```suggestion
    private final UserService userService =
```